src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -75,7 +75,7 @@ public ArgumentList(int level, List<SimpleVariableInformation> args)
 
     internal class VariableInformation : IVariableInformation
     {
-        public string Name { get; private set; }
+        public string Name { get; internal set; }
```

Comment:
Was needed to set display name different from the expression.
Should probably change the constructors instead.